src/impress.js

```diff
@@ -6,13 +6,13 @@
  * in modern browsers and inspired by the idea behind prezi.com.
  *
  *
- * Copyright 2011-2012 Bartek Szopka (@bartaz), 2016-2020 Henrik Ingo (@henrikingo)
+ * Copyright 2011-2012 Bartek Szopka (@bartaz), 2016-2023 Henrik Ingo (@henrikingo)
```

Comment:
Thanks!

I was thinking, let's change this to...

   * Copyright 2011-2012 Bartek Szopka (@bartaz), 2016-2023 Henrik Ingo (@henrikingo) and [70+  other contributors](https://github.com/impress/impress.js/graphs/contributors)

And yes, please update the README file(s) too